src/server/server.rs

```diff
@@ -282,13 +287,15 @@ mod tests {
 
         let mut trans = server.transport();
         trans.report_unreachable(RaftMessage::new());
-        assert_eq!(
-            significant_msg_receiver.try_recv().unwrap(),
-            SignificantMsg::Unreachable {
-                region_id: 0,
-                to_peer_id: 0,
-            }
-        );
+        if let StoreMsg::Unreachable {
+            region_id: 0,
+            to_peer_id: 0,
+        } = significant_msg_receiver.try_recv().unwrap()
+        {
+            // nothing to do
+        } else {
+            assert!(false);
```

Comment:
It's meaningless, why not use `assert_eq` or just panic with meaningful message?